### DISCUSSION TITLE
Ente Auth ( Groups )

### DISCUSSION THREAD

**Discussion by Bayleyvilla** [ente-io/ente#1545] [Enhancements]
Add groups so that the keys can be grouped into categories ( Home - Work - Google - Dropbox - etc etc )
Add option to name the categories ( User Defined )
Hide code following refresh 
Add automatic Backups ( Possibly to SD card on mobile device or a cloud storage Google - OneDrive )

**Comment by vishnukvmd** [comment]
Closing in favor of https://github.com/ente-io/ente/discussions/482.
